src/opnsense/mvc/app/models/OPNsense/Monit/Monit.xml

```diff
@@ -47,6 +47,23 @@
             <default>0</default>
             <Required>Y</Required>
          </ssl>
+         <sslversion type="OptionField">
+            <default>auto</default>
+            <Required>Y</Required>
+            <OptionValues>
+               <auto>AUTO</auto>
+               <sslv2>SSLV2</sslv2>
```

Comment:
can remove these, SSL2 and SSL3 are dead and hardwired to off in LibreSSL/OpenSSL